spec/README.mediawiki

```diff
@@ -79,1681 +79,42 @@ basis for design.
 
 # Resistant to malicious or manipulative trading. Deliberately disruptive behavior should be disincentivized, and volume manipulation mitigated by pseudo-random epoch-based order matching.
 
+==Sections==
+
 The remainder of this document details the design of the proposed DEX.
 It is intended as a first resource when implementing servers and clients.
 Continuing client-server interoperability will likely depend on subtle aspects
 of the specification laid out in this document, especially the following
 critical sections.
 
-'''[[#Communication_Protocols|Communication Protocols]]''' describes the
+'''[[COMMPROTOCOLS.mediawiki|Communication Protocols]]''' describes the
 messaging protocols and communication layer technologies that are to be used
 for the DEX API.
 
-'''[[#Distributed_Exchange_Design_Fundamentals|Distributed Exchange Design Fundamentals]]'''
+'''[[EXCHANGEFUNDAMENTALS.mediawiki|Distributed Exchange Design Fundamentals]]'''
 describes the notable design aspects that facilitate an exchange service with
 the features described above.
 
-'''[[#Client_Order_Management|Client Order Management]]''' details the different
+'''[[EXCHANGEADMIN.mediawiki|Distributed Exchange Administration]]''' describes
+the tasks required to administer the exchange.
+
+'''[[ACCOUNTS.mediawiki|Client Accounts]]''' details account creation.
+
+'''[[ORDERMANAGEMENT.mediawiki|Client Order Management]]''' details the different
```

Comment:
What do you think about numbering the "chapters"?

Reply:
Numbered with brackets